packages/components-css/card/index.scss

```diff
@@ -0,0 +1,22 @@
+.frameless-card {
+  --utrecht-heading-2-font-size: var(--frameless-card-font-size);
```

Comment:
`--frameless-card-font-size` is niet duidelijk of dit behoort tot de titel of de body van de kaart. Misschien wordt het zoiets als `--frameless-card-title-1-font-size`, etc.

Reply:
De font size hoort niet perse bij de card, iedereen kan dit gebruiken maar in de Json bestand zet ik dit in mijn card en om dit aanteroepen moet ik dit `var(--frameless-card-font-size)` zo hebben staan

Reply:
Volgens mij is de card component geen [atoms component](https://atomicdesign.bradfrost.com/chapter-2/#atoms). Dat betekent dat deze component een header en body heeft en misschien later een link. Dus de naam is niet duidelijk gebaseerd op de Design-systeem understanding.